portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -81,13 +95,19 @@ definition {
 	}
 
 	macro deleteABTest {
-		Click(locator1 = "ABTest#KEBAB_AB_TEST");
+		if (${deleteWay} == "kebab") {
```

Comment:
Hey @manuferreira, I think here you could check if the modal is not present (`Modal#MODAL_CONTENT`), If it is not then it would do the `IF` actions and if it already was then it would jump to interact with the modal. What do you think?

I will update this in my next PR. I think this way would be better so that the person who is going to use this macro does not need to understand what they need to pass to the macro depending on the situation they are in. It would only use the macro and regardless of the test situation, the macro would work